package.json

```diff
@@ -27,6 +27,7 @@
     "packages-version": "lerna version --allow-branch main --no-push --no-private -m \"chore(release): Bump package versions\"",
     "release": "npm run release --workspace mongodb-compass --",
     "release-evergreen": "npm run release-evergreen --workspace=mongodb-compass --",
+    "build-dist-package": "rm -Rf packages/compass/dist && DEBUG='hadron*' HADRON_SKIP_INSTALLER='true' HADRON_PRODUCT='mongodb-compass' HADRON_PRODUCT_NAME='MongoDB Compass' HADRON_DISTRIBUTION='compass' npm run release-evergreen",
```

Comment:
This `rm -rf` will remove other builds in dist if you have any, might be better to avoid that maybe? Build process cleans up all the used paths itself anyway IIRC